docs/source/babs-init.rst

```diff
@@ -59,7 +59,8 @@ from a predefined pool by BABS. Examples are ``BIDS``, ``freesurfer``.
        ``freesurfer`` results zip files into current BABS project, then you should name this input
```

Comment:
A couple notes about points 1. and 2. under Specific Restrictions:
- Input datasets don't have to be BIDS datasets, they can also be BIDS derivative datasets.
- How do you need to name your input datasets if there are multiple inputs and one or more isn't zipped?